README.md

```diff
@@ -6,7 +6,7 @@ Shared classes and tests for fog providers and services.
 
 ## Ruby version
 
-Fog-core requires Ruby `2.0.0` or later.
+`fog-core` requires Ruby `2.5.0` or later.
```

Comment:
Wait, this is a huge change.

I recall that we had to wait a long time before dropping ruby `1.9` because of LTS distros that still had support for it.

Is there a reason (besides the obvious non maintained ruby version) for this change?

Reply:
Maybe @geemus can chime in since he was the one who told me to wait back then?

Reply:
I checked https://www.ruby-lang.org/en/downloads/branches/
```
status: eol
release date: 2019-12-25
normal maintenance until: 2022-04-01
EOL: 2023-03-31
Ruby 2.6

status: eol
release date: 2018-12-25
normal maintenance until: 2021-04-01
EOL: 2022-04-12
Ruby 2.5

status: eol
release date: 2017-12-25
normal maintenance until: 2020-04-01
EOL: 2021-04-05
```

I thought that older versions of ruby should not be used. Maybe we should stick to >= 2.7.
Actually, this all about rubocop rules + compatibility code.

Why I chose 2.5? Because fog-core has tests starting from 2.5.
But, if you think, that we *must* stick to 2.0, I can revert this part of changes.

Do you think we still have <2.5 users? 
Should next release in 2k23 (2k24) still support <2.5 ruby?

Reply:
Last time i checked we had a lot of users using old distros in production that had LTS support for older versions of ruby but i dunno how it is nowadays.

If i remember correctly it was CentOS LTS and RedHat LTS that hold us true to ruby 2.0.

Reply:
I guess that since our need to update the version of ruby is just because of rubocop, maybe we could freeze to a specific version of rubocop instead?

At leat for now?

And later we can tackle this on a different PR, with focus on updating ruby version?

Reply:
Ok, LTS distros with 2.0 is a good point. I'll revert this. Anyway, we must keep in mind, that tests actually running under 2.5 and higher.